templates/common.jade

```diff
@@ -0,0 +1,19 @@
+extends /templates/layout
+
+block body
+    if task == "edit"
+        .container#content
+         #relationshipForm
```

Comment:
This line needs indenting once more, so that the `relationshipForm` div is rendered inside the `content` div.